includes/admin/class-wc-admin-status.php

```diff
@@ -339,6 +339,29 @@ private static function log_table_bulk_actions() {
 		}
 	}
 
+	/**
+	 * Prints table info if a base table is not present.
+	 */
+	private static function output_tables_info() {
+		$missing_tables = WC_Install::verify_base_tables( false );
+		if ( 0 === count( $missing_tables ) ) {
+			return;
+		}
+		?>
+
+		<br>
+		<strong style="color:#a00;">
+			<span class="dashicons dashicons-warning"></span>
+			<?php
+				esc_html_e( 'Missing base tables: ', 'woocommerce' );
+				echo esc_html( implode( ', ', $missing_tables ) );
+				esc_html_e( '. Some WooCommerce functionality may not work as expected.', 'woocommerce' );
```

Comment:
This might be difficult to translate? Should this use %s instead?

Reply:
Good catch. Resolved in https://github.com/woocommerce/woocommerce/pull/26395/commits/34e5d3daa39e9a7475eebeafd3f6617722afeb1f.